src/platform/msrCrypto/README.md

```diff
@@ -0,0 +1,7 @@
+This source comes from here:
+https://github.com/microsoft/MSR-JavaScript-Crypto
```

Comment:
For some reason msrCrypto is not an npm package. So we have to copy it. And it's not a module, so it has to be included in the source.